src/pretalx/agenda/templates/agenda/talk.html

```diff
@@ -50,6 +47,13 @@ <h3 class="talk-title">
     </h3>
     <div class="talk row">
         <div class="talk-content col-lg-8 col-md-6 col-xs-12">
+            <ul class="talk-slots">
+                {% for talk in talk_slots %}
+                <li class="talk-slot">
+                    {{ talk.start.date|date:"c" }}, {{ talk.start|date:"H:i" }}–{{ talk.end|date:"H:i" }}, {{ talk.room.name }}
+                </li>
+                {% endfor %}
+            </ul>
```

Comment:
show a list of all slot times/rooms the talk (=submission) is given.

Reply:
While this is correct in general, for talks with only one slot (which will be the default), we'll have a list with only one entry, which looks kinda stupid. I'd prefer the usual format if there is only one talk slot available.